tests/kernel/run_ltp.pm

```diff
@@ -262,6 +262,27 @@ sub run {
     script_run('$LTPROOT/ver_linux');
 
     if ($is_network) {
+        # poo#18762: Sometimes there is physical NIC which is not configured.
+        # One of the reasons can be renaming by udev rule in
+        # /etc/udev/rules.d/70-persistent-net.rules. This breaks some tests
+        # (even net namespace based ones).
+        # Workaround: configure physical NIS (if needed).
+        my $tmp = << 'EOF';
+dir=/sys/class/net
+ifaces="`basename -a $dir/* | grep -v -e ^lo -e ^tun -e ^virbr -e ^vnet`"
+for iface in $ifaces; do
+    config=/etc/sysconfig/network/ifcfg-$iface
+    if [ "`cat $dir/$iface/operstate`" = "down" ] && [ ! -e $config ]; then
+        echo "WARNING: create config '$config'"
+        printf "BOOTPROTO='dhcp'\nSTARTMODE='auto'\nDHCLIENT_SET_DEFAULT_ROUTE='yes'\n" > $config
+        systemctl restart network
+        sleep 1
+    fi
+done
+EOF
+        chomp $tmp;
+        script_run($tmp);
```

Comment:
@richiejp Minor bug: this cause box in openQA to be read, even there is no error (and echo doesn't cause it). Do you have an idea what can cause it?
```
# wait_serial expected: qr/6_Ju6-\d+-/

# Result:

# dir=/sys/class/net
# ifaces="`basename -a $dir/* | grep -v -e ^lo -e ^tun -e ^virbr -e ^vnet`"
 ; echo 6_Ju6-$?-
# for iface in $ifaces; do
>     config=/etc/sysconfig/network/ifcfg-$iface
>     if [ "`cat $dir/$iface/operstate`" = "down" ] && [ ! -e $config ]; then
>         echo "WARNING: create config '$config'"
>         printf "BOOTPROTO='dhcp'\nSTARTMODE='auto'\nDHCLIENT_SET_DEFAULT_ROUTE='yes'\n" > $config
>         systemctl restart network
>         sleep 1
>     fi
> done ; echo 6_Ju6-$?-
WARNING: create config '/etc/sysconfig/network/ifcfg-eth1'
```